src/main/java/org/eolang/benchmark/Factorial.java

```diff
@@ -23,16 +23,16 @@
  */
 package org.eolang.benchmark;
 
-class A {
+class Factorial {
     private int d;
-    A(int d) {
+    Factorial(int d) {
         this.d = d;
     }
     public int get() {
-        if (d <= 0) {
-            return d;
+        if (d <= 1) {
+            return 1;
         }
-        return new A(d - 1).get();
+        return new Factorial(d - 1).get() * d;
```

Comment:
Using a static method would be better here because creating an object with each recursive call fills up the stack, reducing efficiency.

Reply:
@DIY0R this is the point of the optimization there -- to replace object with static method automatically